### PR TITLE
feat: Profile sidebar improvements

### LINKED ISSUE #9096: Profile sidebar improvements
lets solve some issues here 

image

design: https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=145-4743&mode=design&t=RpDGEvxjL3QU0BUX-11

## Issues with previous design
- small icons
- !bottom part moving! - this was not super pleasant when opening the sidebar fast and wanting to access a item in the navigation
- profile view is used quite often, but the button is quite small 

## Changes
- from top to bottom 

### top part 
image
- view profile button stroke is gone, the whole green part is clickable - cursor pointer, when clicked, user redirected to profile page 

image

disconnect and copy icons are now next to each other

### Assets 
- no change here

### Bottom navigation and links 
image
- bottom part is fixed
- where? please put it, so when there is full section (recent nfts, 4 assets rows) its 16px from the true black divider. 

image

- it will stay like this for all possible options (less asset rows, or no recent nfts)

image

like this 

let me know if anything

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #9096

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-01-24 at 11 13 39

CleanShot 2024-01-24 at 11 14 44

### CODE REVIEW

File: components/common/ConnectWallet/WalletAssetIdentity.vue [hunk 1]
```diff
@@ -1,16 +1,32 @@
...
-    <IdentityItem
-      :account="account"
-      :label="display || shortenedAddress"
-      :prefix="urlPrefix"
-      variant="button" />
-    <div class="flex flex-col">
+    <NuxtLink
+      :to="`/${urlPrefix}/u/${account}`"
+      class="w-full"
+      @click="closeModal">
```
Comment (roiLeo) [inline components/common/ConnectWallet/WalletAssetIdentity.vue:6]: why wrap it in `NuxtLink` as `IdentityItem` is already doing likewise

File: components/common/ConnectWallet/WalletAssetIdentity.vue [hunk 2]
```diff
@@ -1,16 +1,32 @@
...
+        :label="display || shortenedAddress"
+        :prefix="urlPrefix">
+        <template #default="{ label }">
+          <div class="pl-3">
+            <div class="font-bold mb-1">
+              {{ label }}
+            </div>
+
+            <div class="text-xs text-k-grey">
+              {{ $t('viewProfile') }}
```
Comment (roiLeo) [inline components/common/ConnectWallet/WalletAssetIdentity.vue:18]: already default value in `IdentityItem`

**Review by roiLeo — APPROVED** [review]
lgtm, improvements

File: components/common/ConnectWallet/WalletAssetIdentity.vue (hunk 1 above)
Comment (hassnian) [reply components/common/ConnectWallet/WalletAssetIdentity.vue:6]: green section needs to be clickable. 

CleanShot 2024-01-24 at 13 41 48@2x

File: components/common/ConnectWallet/WalletAssetIdentity.vue (hunk 2 above)
Comment (hassnian) [reply components/common/ConnectWallet/WalletAssetIdentity.vue:18]: removed the "variant" prop from `IdentityItem` , from what I could see that wasn't used elsewhere.

now is using a slot , passing this "variant", so no default prop, do you think I'd be better to have Inside , like before ?


CleanShot 2024-01-24 at 13 41 21@2x

Comment (roiLeo) [reply components/common/ConnectWallet/WalletAssetIdentity.vue:18]: yes! Imo it's better to use `slot` and refactor component when we have unused stuff

File: components/common/ConnectWallet/WalletAssetIdentity.vue (hunk 1 above)
Comment (roiLeo) [reply components/common/ConnectWallet/WalletAssetIdentity.vue:6]: Alright design has changed

**Review by preschian — APPROVED** [review]

**Comment by yangwao** [comment]
Thanks!
pay 50 usd

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 6.41 USD/DOT ~ 7.8 $DOT
🧗 [16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD](https://polkadot.subscan.io/account/16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)
🔗 [0xc104d69893a3594e4cfc082bee9a408b0a6dd0261d366c6975867fa70ae62235](https://polkadot.subscan.io/extrinsic/0xc104d69893a3594e4cfc082bee9a408b0a6dd0261d366c6975867fa70ae62235)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**